pkg/credentialquery/credentialquery.go

```diff
@@ -80,10 +80,10 @@ func (c *Instance) GetSubmissionRequirements(
 		return nil, err
 	}
 
-	// TODO: https://github.com/trustbloc/wallet-sdk/issues/165 remove this code after to re enable Schema check.
-	for i := range query.InputDescriptors {
-		query.InputDescriptors[i].Schema = nil
-	}
+	//// TODO: https://github.com/trustbloc/wallet-sdk/issues/165 remove this code after to re enable Schema check.
+	// for i := range query.InputDescriptors {
+	//	 query.InputDescriptors[i].Schema = nil
+	// }
```

Comment:
If we still need this code snippet for now, should line 83 just have `//` instead of `////`? Seems unnecessary

Reply:
Oops, I meant to remove that. Good catch! Fixed.